app/views/events/_form.html.erb

```diff
@@ -28,11 +28,7 @@
 
   <div class="field">
     <%= f.label :occurs_at %> <small class="vermelho">*</small><br />
-    <% if @event.new_record? %>
-      <%= f.text_field :occurs_at, :class => 'datepicker' %>
-    <% else -%>
-      <%= f.text_field :occurs_at, :class => 'datepicker', :value => l(@event.occurs_at, :format => :picker) %>
-    <% end %>
+    <%= f.text_field :occurs_at, class: 'datepicker' %>
```

Comment:
you are right. I supposed it was used in another place but isn't.
already removed